dashboard/app/lib/javascripts/dashboard/main.js

```diff
@@ -38,6 +38,11 @@ window.Dashboard = {
 			dispatcher: this.Dispatcher,
 			trigger: false
 		});
+
+		this.client.ping().then(function (args) {
+			console.log(args)
+		});
```

Comment:
A better place for this is [in the `CONFIG_READY` event handler below](https://github.com/flynn/flynn/blob/master/dashboard/app/lib/javascripts/dashboard/main.js#L97-L103).